Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -550,6 +578,48 @@ - (void)configureCell:(FSCalendarCell *)cell
     }
 }
 
++ (UIColor *)fillColorForDateStyle:(BPKCalendarDateCellStyle)style
+                        appearance:(FSCalendarAppearance *)appearance {
+    switch (style) {
+        case BPKCalendarDateCellStyleNormal:
+            return appearance.borderDefaultColor;
+        case BPKCalendarDateCellStylePositive:
+            return BPKColor.glencoe;
+        case BPKCalendarDateCellStyleNeutral:
+            return BPKColor.erfoud;
+        case BPKCalendarDateCellStyleNegative:
+            return BPKColor.hillier;
+        case BPKCalendarDateCellStyleCustom:
+            NSAssert(NO, @"Don't call `fillColorForDateStyle:appearance:` with the custom cell style");
```

Comment:
This will only be called if we do something wrong in Backpack, right?

Reply:
Yes it's to prevent and alert us if we do make such a mistake

Reply:
I wonder if it should be more descriptive

Something like
`fillColorForDateStyle:appearance:` should not be called with the custom cell style. For a custom cell style, `calendar:fillColourForDate:` and `calendar:textColourForDate:` should be used instead.

Reply:
I suppose it could

Reply:
The important thing is that it crashes to alert the developer that they are about to create a bug

Reply:
See 157fb04